projects/canopy/src/lib/forms/input/input-field.component.scss

```diff
@@ -1,9 +1,115 @@
 @import '../../../styles/mixins';
-@import './input.scss';
+
+:root {
+  --suffix-margin: var(--space-sm);
+  --suffix-button-margin: 0.3125rem;
+}
 
 .lg-input-field {
+  $block: &;
```

Comment:
I went for block as in the block in**B**EM, but in hindsight that may be misread, did you read it as block like `display:block`?

Reply:
Yeah it took me a couple of takes to realise what was going on, maybe renaming it could make the code easier to scan. Then again, it's pretty clear once you know what's going on. So happy either way really :)